docs/Reference/GLOSSARY.md

```diff
@@ -2,60 +2,92 @@
+credentials defined by the [W3C Verifiable Credentials specification](https://www.w3.org/TR/did-core/#bib-vc-data-model)
+
+### Verifiable data registry (VDR)
+
+A system that facilitates the creation, verification, updating, and/or deactivation of decentralized identifiers and [DID](#decentralized-identifier-did)
+documents. A verifiable data registry might also be used for other cryptographically-verifiable data structures such as
+verifiable credentials.
+
+### Verifiable presentation (VP)
 
-Verifiable presentation
-: Verifiable presentation (VP)
+A verifiable presentation (VP) is a tamper-evident presentation encoded in such a way that authorship of the data can be
+trusted after a process of cryptographic verification
 
-VASP
-: FATF defines a virtual asset service provider (VASP) as the following: *“Any natural/legal person who ... as a business conducts one or more of the
-following activities or operations for or on behalf of another natural or legal person: i. exchange between virtual
-assets and fiat currencies; ii. exchange between one or more forms of virtual assets; iii. transfer of virtual assets;
-iv. safekeeping and/or administration of virtual assets or instruments enabling control over virtual assets; and v.
-participation in and provision of financial services related to an issuer’s offer and/or sale of a virtual
-asset.”*[FATF guidance to the G20](https://www.fatf-gafi.org/publications/virtualassets/documents/report-g20-so-called-stablecoins-june-2020.html)
+### Virtual Asset Service Provider (VASP)
 
+FATF defines a virtual asset service provider (VASP) as the following:
+“Any natural/legal person who ... as a business conducts one or more of the following activities or operations for or on
+behalf of another natural or legal person:
+i. exchange between virtual assets and fiat currencies; ii. exchange between one or more forms of virtual assets; iii.
+transfer of virtual assets; iv. safekeeping and/or administration of virtual assets or instruments enabling control over
+virtual assets; and v. participation in and provision of financial services related to an issuer’s offer and/or sale of
+a virtual
+asset.” [FATF guidance to the G20](https://www.fatf-gafi.org/publications/virtualassets/documents/report-g20-so-called-stablecoins-june-2020.html)
 
+## Z
 
+### Zero-knowledge Proof (ZKP)
 
+In cryptography, a zero-knowledge proof or zero-knowledge protocol is a method by which one party (the prover) can prove to another party (the verifier) that a given statement is true, without conveying any information apart from the fact that the statement is indeed true. The essence of zero-knowledge proofs is that it is trivial to prove that one possesses knowledge of certain information by simply revealing it; the challenge is to prove such possession without revealing the information itself or any additional information ([wikipedia](https://en.wikipedia.org/wiki/Zero-knowledge_proof)).
```

Comment:
```suggestion
In cryptography, a zero-knowledge proof or zero-knowledge protocol is a method by which one party (the prover) can prove to another party (the verifier) that a given statement is true, without conveying any information apart from the fact that the statement is indeed true. The essence of zero-knowledge proofs is that it is trivial to prove that one possesses knowledge of certain information by simply revealing it; the challenge is to prove such possession without revealing the information itself or any additional information ([Zero-knowledge proof on wikipedia](https://en.wikipedia.org/wiki/Zero-knowledge_proof)).
```

Reply:
descriptive link text must be the title of the page you land on, wdyt about this construction? (Wikipedia was too vague)